charts/jenkins/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 description: The leading open source automation server
-version: 1.5.1
+version: 1.5.2
```

Comment:
Please bump the minor version to follow semver.

Reply:
Hi,

sry, i don't understand whats wrong with the 1.5.2?

Regards
Sascha

Reply:
Following semver, a new feature requires a minor version bump.

Reply:
Ah, sry so a 1.6.1 would be correct.

Reply:
1.6.0 would be the next minor version.

Reply:
oh man, i'm sorry...maybe i should drink more coffee before i do things like that...